apps/src/applab/api.js

```diff
@@ -448,6 +448,13 @@ exports.deleteRecord = function (table, record, onComplete, onError) {
                            'onError': onError});
 };
 
+exports.onRecordEvent = function (table, onRecord) {
+  return Applab.executeCmd(null,
+                           'onRecordEvent',
+                           {'table': table,
+                            'onRecord': onRecord});
```

Comment:
Just curious, what would happen if I passed the wrong type of argument to onRecord?  (Would an error be reported right away or after the event fired where it would be less clear what happened?)